system-int-test/test/cmr/system_int_test/ingest/generics_test.clj

```diff
@@ -103,8 +81,8 @@
    interface. Use the same native-id for all these steps"
 
   (let [native-id (format "Generic-Test-%s" (UUID/randomUUID))
-        generic-requester (partial generic-request "grid" "PROV1" native-id)
-        good-generic-requester (partial generic-requester grid-good)]
+        generic-requester (partial gen-util/generic-request-with-echo-token "grid" "PROV1" native-id)
```

Comment:
Should this still be echo token?

Reply:
i am not sure, but that change is not in scope for this ticket i believe. this group of generic ingest tests was already using echo token, I just renamed it to be explicit about what it is doing. and (previously) refactored the function to be able to accept a custom token as well. 

in other words, previously:
this function used echo token, another function in providers_ingest_test used custom tokens.
now:
both groups of tests use the same function which expects a token parameter. 'generic-request-with-echo-token' is simply a partial, which defines that first parameter to be an echo token.

(see gen-util ns here, and/or pr#1673 for cmr-8380)